GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -501,10 +501,13 @@ extension ChatViewModel {
 
             pendingMessages.append(pendingMessage)
 
-            let queueItem = ChatItem(kind: .queueOperator)
+            // Avoid showing enqueud operator according to parameter.
+            if !shouldSkipEnqueueingState {
+                let queueItem = ChatItem(kind: .queueOperator)
+                queueOperatorSection.set([queueItem])
+                action?(.refreshSection(2))
```

Comment:
Weird to see the number 2 just like that

Reply:
That how it was initially before I wrapped it in `if !shouldSkipEnqueueingState` condition. Updated ced3a40255b8defb13022647300adf1e3cdae6a2.